sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/tests/SessionRecords/AccountCollectionTests/CreateOrUpdate().json

```diff
@@ -92,7 +92,7 @@
       }
     },
     {
-      "RequestUri": "https://management.azure.com/subscriptions/cf65b9a6-fe0f-4011-881c-aba5a5fb8603/resourceGroups/testRg-3726/providers/Microsoft.DeviceUpdate/accounts/Account-7741?api-version=2022-04-01-preview",
+      "RequestUri": "https://management.azure.com/subscriptions/cf65b9a6-fe0f-4011-881c-aba5a5fb8603/resourceGroups/testRg-3726/providers/Microsoft.DeviceUpdate/accounts/Account-7741?api-version=2022-10-01",
```

Comment:
It looks like that only the api-version is updated in all the test records, did we only replace the old version with the new version or did we re-record all the tests? If it is the first case, I think we should re-record all the tests to ensure the service works as expected. You can see this [guide](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#running-tests) and set the `AZURE_TEST_MODE` to `Record` to record all the tests.